src/server/middleware/csp.js

```diff
@@ -65,7 +65,12 @@ const csp = () => (req, res, next) => {
     updatedPolicy = insertSource(
       updatedScriptSrc,
       'connect-src',
-      `${ip.address()}:${process.env.HTTP_ONE_APP_DEV_CDN_PORT} localhost:${process.env.HTTP_ONE_APP_DEV_CDN_PORT}`
+      [
+        `${ip.address()}:${process.env.HTTP_ONE_APP_DEV_CDN_PORT}`,
+        `localhost:${process.env.HTTP_ONE_APP_DEV_CDN_PORT}`,
+        `${ip.address()}:${process.env.HTTP_ONE_APP_DEV_PROXY_SERVER_PORT}`,
+        `localhost:${process.env.HTTP_ONE_APP_DEV_PROXY_SERVER_PORT}`,
```

Comment:
maybe we should not be so opinionated on the ports during development.
@anescobar1991 thoughts on :
```suggestion
        `${ip.address()}:*`,
        'localhost:*',
```

Reply:
As long as its just for development I dont see any issues. 👍

Reply:
Although maybe this would confuse users who dont realize that `localhost` is not just open and when they get to production they would expect it to work but it doesnt?

Reply:
i think that might be the case anyway by opening up some ports

Reply:
@enuchi would you be able to make this change and then i think this is good to go.

Reply:
but those ports are reserved for one app usage by nature of them being env vars used by one app though no?

Reply:
thats a fair point, so maybe leave as is and open it up later if we find a need

Reply:
yeah! if we do the opposite it becomes a breaking change!